.github/dependabot.yml

```diff
@@ -1,13 +1,22 @@
+# This file is only used for vulnerability alerts, not for automatic updates.
```

Comment:
You can remove this file, dependabot is only useful for NPM security patches, the rest can be patched by Renovate.